config/crd/bases/etcdcluster.cluster.x-k8s.io_etcdadmclusters.yaml

```diff
@@ -524,6 +524,10 @@ spec:
                       description: File defines the input for generating write_files
                         in cloud-init.
```

Comment:
was this just not checked in before?

Reply:
Good point. Let me look into it. It could be that this is reusing an upstream cluster-api struct, so when the capi dependency got updated, this CRD inherited new fields that capi added to their API.

Will update the PR description with what I find.

/hold

Reply:
So those fields were added to the etcdam-bootstrap-providera bit ago: https://github.com/aws/etcdadm-bootstrap-provider/pull/17
That struct (etcdadmconfig) it's imported from that repo into this CRD.

It seems we never regenerated the manifests here. So I guess this is expected